src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,46 @@
+package duke;
+
+import java.time.LocalDate;
+
+/**
+ * Represents a specific type of Task with an additional deadline(ddl) field.
+ */
+public class Deadline extends Task {
+    private LocalDate ddl;
```

Comment:
Is there any reason why this variable is ddl? Perhaps it will be better if it can be changed to a name that explains the name entiry.

Reply:
Thanks for pointing this out. I have changed the variable name to deadline in the latest commit!